ckanext/who_afro/assets/css/who-afro.css

```diff
@@ -615,19 +615,25 @@ p.small {
     margin-bottom: 0;
 }
 
-.promoted .big-search-form .lens-btn {
+.promoted .big-search-form .lens-btn,
+.promoted .big-search-form .lens-btn:hover,
+.insights .insight-row .search-section .big-search-form .lens-btn,
+.insights .insight-row .search-section .big-search-form .lens-btn:hover {
+    border-color: var(--primary-navy) thin 1px !important;
```

Comment:
`!important` is obviously a big red flag and should really be a last resort.  Is it really not obvious how else this could be done?

Reply:
Ah wait, this is from Toavina? Fine.  If you can see how to remove it in 3-5 mins, then do it, but otherwise just leave it.